src/main/java/com/commercetools/project/sync/service/impl/CustomObjectServiceImpl.java

```diff
@@ -90,11 +99,28 @@ public CustomObjectServiceImpl(@Nonnull final SphereClient sphereClient) {
     return createCustomObject(lastSyncCustomObjectDraft);
   }
 
+  @Override
+  public CustomObjectService attachRunnerName(String runnerName) {
```

Comment:
Please use `final` for the parameter. Also please use `@Nonnull` or `@Nullable` as it makes it easier to know if it could be passed as `null` or not.

Reply:
a5b7531c545f6f905e4e58d8bab4838fab0c8eea